yazi-config/preset/yazi.toml

```diff
@@ -139,7 +139,7 @@ cd_origin = "top-center"
 cd_offset = [ 0, 2, 50, 3 ]
 
 # create
-create_title  = "Create:"
+create_title  = [ "Create:", "Create (dir):" ]
```

Comment:
This is a breaking change, which will cause a crash for users whose existing config contains `create_title = "Create:"`, as the type has changed, and Yazi can't parse it.

Please support both array and string formats, and print a warning when a string is detected to prompt users to switch to the new array format.